src/Nethermind/Nethermind.Evm.Test/TransactionProcessorFeeTests.cs

```diff
@@ -106,6 +106,32 @@ public void Check_paid_fees_multiple_transactions(bool withFeeCollector)
         tracer.BurntFees.Should().Be(84000);
     }
 
+    [TestCase(false)]
+    [TestCase(true)]
+    public void Check_paid_fees_with_blob(bool withFeeCollector)
```

Comment:
Could you also confirm with a test that the code adds balance, not just sets it?